lib/skate/detours/detours.ex

```diff
@@ -5,54 +5,120 @@ defmodule Skate.Detours.Detours do
 
   import Ecto.Query, warn: false
   alias Skate.Repo
-
-  alias Skate.Detours.Db.Detour
+  alias Schedule.Gtfs.Direction
+  alias Skate.Detours.Db.Detour, as: DbDetour
```

Comment:
Is there a struct called `Detour` that will run into a naming collision with this? Or is this mainly for clarity? [The other example](https://github.com/mbta/skate/blob/445884d91d766641b165a0a02af345f13696929a/lib/skate/settings/test_group.ex#L4) where I've seen this kind of this is in `TestGroup`, but the parent module and the DB module were both named `TestGroup`, so it was necessary.

Reply:
Totally right! At one point, I thought an error I was seeing was a typing issue, so I was pulling in a Detour model that looked a little different. Later became unnecessary, so rolled this change back